### DISCUSSION TITLE
Property 'resend' does not exist on type 'SupabaseAuthClient'.

### DISCUSSION THREAD

**Discussion by neeia** [supabase/supabase#28829] [Questions] [answered]
As per [the documentation](https://supabase.com/docs/reference/javascript/auth-resend), I wrote the following code below:

` const { error } = await supabase.auth.resend({ type: 'signup', email: email })`

However, this results in a ts error saying that the the auth client has no such property. This is a browser supabase client, created using `createClient`. My Supabase project requires email verification. I have restarted the ts server and the error still persists. Has anyone been able to resend verification emails? Does anyone have an idea as to how I could go about fixing this?

**Comment by GaryAustin1** [comment, marked as answer]
What version of supabase-js are you using?

**Reply by neeia** [reply]
I'm using v2.12.1.

**Reply by neeia** [reply]
Updating to the latest version fixed the issue, thanks! Not sure why I didn't try it previously.
